tests/utils/vmgroup/vmgrouplifecycle.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This util is going to hold various helper methods related to vmgroup to be consumed by testcases.
+// vmgroup creation, deletion is supported as of now.
+
+
+package vmgroup
+
+import (
+    "log"
+
+    "github.com/vmware/docker-volume-vsphere/tests/constants/admincli"
+    "github.com/vmware/docker-volume-vsphere/tests/utils/ssh"
+)
+
+
+// CreateVmgroup method is going to create a vmgroup and adds vm to it
+func CreateVmgroup(ip, name string, vmName string) ([]byte, error) {
```

Comment:
`ip, name string, vmName string` => `ip, name, vmName string` .. to keep it consistent